zstackctl/zstackctl/ctl.py

```
@@ -775,6 +775,45 @@ def start_remote_mn( host_post_info):
 
+
+def run_mysqlcheck_zstack(db_user, db_password, db_hostname, db_port):
+    error_if_tool_is_missing('mysqlcheck')
+    try:
+        if db_password:
+            shell('mysqlcheck -u %s -p%s --host %s --port %s zstack' % (
+                shell_quote(db_user), shell_quote(db_password), shell_quote(db_hostname), shell_quote(db_port)))
+        else:
+            shell('mysqlcheck -u %s --host %s --port %s zstack' % (
+                shell_quote(db_user), shell_quote(db_hostname), shell_quote(db_port)))
+    except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)